format.sh

```diff
@@ -34,6 +34,12 @@ else
     fi
 fi
 
+if [ ! $(command -v clang-tidy) ]
```

Comment:
I'm illiterate in bash but it should probably prioritize `clang-tidy-$FORMAT_VER` like for clang-format ?

Or add `TIDY_VER` ?

Reply:
There is no difference or reason to prioritize a specific version for clang-tidy afaik, so it seems unnecessary to me. It's a different story for clang-format because we want to prioritize using clang-format-11 instead of the default version for whatever package manager you're using.

Reply:
Ok!